src/apis/comment.ts

```diff
@@ -0,0 +1,24 @@
+import { axiosInterface } from './axios';
+
+type CommentData = {
+  comment: string;
+  postId: string;
+};
+export const sendComment = async ({ comment, postId }: CommentData) => {
+  try {
+    await axiosInterface.post('/comments/create', {
+      comment,
+      postId,
+    });
+  } catch (error) {
+    console.error(error);
+  }
```

Comment:
```suggestion
  } catch (error) {
    if(error instanceof AxiosError){
       console.error(error.message)
    } else if (error instanceof Error) {
      console.error(error.message)
    } else {
      console.error(error)
    }
  }
```

저는 이런식으로 처리해볼까 하고 있는데 어떤가요..? 이거 그냥 여기서 하드코딩 한거라 suggestion 하면 큰일납니다 허허🤣